st3/lsp_utils/__init__.py

```diff
@@ -1,4 +1,5 @@
 from ._client_handler import ClientHandler
+from ._client_handler import as_notification_handler, as_request_handler
```

Comment:
I'm now used to the convention where each import is separate (apart from types).